Description: Upstream changes introduced in version 4.12.0-0yavdr0~oneiricubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 yavdr-unity-2d (4.12.0-0yavdr0~oneiricubuntu1) oneiric; urgency=low
 .
   * removed thrashcan from launcher
 .
 The person named in the Author field signed this changelog entry.
Author: Gerald Dachs <gda@dachsweb.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- yavdr-unity-2d-4.12.0.orig/launcher/Launcher.qml
+++ yavdr-unity-2d-4.12.0/launcher/Launcher.qml
@@ -135,7 +135,7 @@ LauncherDropItem {
         items.appendModel(applications);
         items.appendModel(workspaces);
         items.appendModel(devices);
-        shelfItems.appendModel(trashes);
+/*        shelfItems.appendModel(trashes); */
     }
 
     Connections {
